apps/cowswap-frontend/src/legacy/state/orders/actions.ts

```diff
@@ -36,7 +36,7 @@ export const CREATING_STATES = [OrderStatus.PRESIGNATURE_PENDING, OrderStatus.CR
 //  - Derived/additional information that is handy for this app
 // Doesn't have input/output tokens, these are declared in the subtypes of this base type
 // includes sellAmountBeforeFee as this is required for checking unfillable orders
-export interface BaseOrder extends Omit<OrderCreation, 'signingScheme'> {
+export interface BaseOrder extends OrderCreation {
```

Comment:
No longer omitting `signingScheme` from the order obj so we can tell whether it's an offchain order